IdentityCore/src/MSIDOAuth2Constants.m

```diff
@@ -108,3 +107,17 @@
 NSString *const MSID_LEGACY_TOKEN_CACHE_TYPE             = @"legacysingleresourcetoken";
 NSString *const MSID_ID_TOKEN_CACHE_TYPE                 = @"idtoken";
 NSString *const MSID_GENERAL_TOKEN_CACHE_TYPE            = @"token";
+
+NSString *MSID_OAUTH2_AUTHORIZE_SUFFIX(void)
+{
+    __auto_type apiVersion = MSIDConfiguration.defaultConfiguration.aadApiVersion;
+
+    return [NSString stringWithFormat:@"/oauth2/%@%@authorize", apiVersion ?: @"", apiVersion ? @"/" : @""];
```

Comment:
ternary within a ternary is a hard read.

Reply:
changed